### ISSUE TITLE
Error when using Julia 1.0.0

### ISSUE THREAD

**Issue by davidmanzanares** [timholy/ProfileView.jl#98]
It seems that it fails to compile with
```
julia> using ProfileView
[ Info: Precompiling ProfileView [c46f51b8-102a-5cf2-8d2c-8597cb0e0da7]
ERROR: LoadError: syntax: extra token "TagData" after end of expression
Stacktrace:
 [1] include at ./boot.jl:317 [inlined]
 [2] include_relative(::Module, ::String) at ./loading.jl:1038
 [3] include(::Module, ::String) at ./sysimg.jl:29
 [4] top-level scope at none:2
 [5] eval at ./boot.jl:319 [inlined]
 [6] eval(::Expr) at ./client.jl:389
 [7] top-level scope at ./none:3
in expression starting at /home/dv/.julia/packages/ProfileView/Ug9WA/src/ProfileView.jl:17
ERROR: Failed to precompile ProfileView [c46f51b8-102a-5cf2-8d2c-8597cb0e0da7] to /home/dv/.julia/compiled/v1.0/ProfileView/GJsyc.ji.
Stacktrace:
 [1] error(::String) at ./error.jl:33
 [2] macro expansion at ./logging.jl:313 [inlined]
 [3] compilecache(::Base.PkgId, ::String) at ./loading.jl:1184
 [4] _require(::Base.PkgId) at ./logging.jl:311
 [5] require(::Base.PkgId) at ./loading.jl:852
 [6] macro expansion at ./logging.jl:311 [inlined]
 [7] require(::Module, ::Symbol) at ./loading.jl:834
```

Thank you :slightly_smiling_face:

**Comment by timholy** [comment]
GtkReactive just got tagged today, so I predict it won't be long now.

**Comment by timholy** [comment]
Fixed by #99

**Comment by davidmanzanares** [comment]
Thank you! Works fine now!
